src/WordsUI.java

```diff
@@ -137,4 +223,59 @@ public void actionPerformed(ActionEvent e) {
 		window.setVisible(true);
 		window.setResizable(false);
 	}
+	
+	/**
+	 * Clears all objects from the UI.  After calling this method, the user will see only an empty grid.
+	 * Typically, the caller will want to immediately repopulate the grid with new objects.
+	 */
+	public void clear() {
+		content.clear();
+		grid.repaint();
+	}
+	
+	/**
+	 * Adds an object to the UI at a given position.
+	 * 
+	 * @param x The x coordinate of the cell to place the object
+	 * @param y The y coordinate of the cell to place the object
+	 * @param className The name of the class of this object
+	 * @param objName The name of the object
+	 * @param message The message that the object should say.  May be null.
+	 */	
+	public void add(int x, int y, String className, String objName, String message) {
```

Comment:
We have a class ```WordsPosition``` that contains x and y. Can we use that in this interface? I think it'll be cleaner: for each object, we pass in the objects that it holds.